tg/controllers.py

```diff
@@ -663,8 +668,14 @@ def render(self) -> None:
         self.queue.put(self._render)
 
     def _render(self) -> None:
-        self.render_chats()
-        self.render_msgs()
+        self._render_chats()
+        self._render_msgs()
+        self._render_status()
```

Comment:
that is incorrect. it creates unexpected behavior when sending msgs with `i`. each render invocation removes msg entered. 
so, status line shouldn't be updated during msg inserting.